Add final blog post and link to skeleton post (#30)
jennaswa/MDAnalysis.github.io: _posts/2024-09-04-outreachy2024_adetutudeborah.md

---
layout: post
title: "Outreachy 2024 - Develop a communications strategy for a growing MDAnalysis user and contributor base"
---

## Goals of the Project

> The goal of this project is to develop a comprehensive social media and communications strategy to grow MDAnalysis user and contributor base. This involves conducting surveys with MDAnalysis users and developers to understand which communication channels they prefer to use to seek information about and engage with the MDAnalysis project. It also entails finding ways to boost participation and involvement from everyone in the MDAnalysis community and to also implement the setup of the MDAnalysis newsletter.

Reviewer: ```suggestion
The goal of this project was to develop a comprehensive social media and communications strategy to grow MDAnalysis user and contributor base. This involves conducting surveys with MDAnalysis users and developers to understand which communication channels they prefer to use to seek information about and engage with the MDAnalysis project. It also entails finding ways to boost participation and involvement from everyone in the MDAnalysis community and to also implement the setup of the MDAnalysis newsletter.
```

> The goal of this project is to develop a comprehensive social media and communications strategy to grow MDAnalysis user and contributor base. This involves conducting surveys with MDAnalysis users and developers to understand which communication channels they prefer to use to seek information about and engage with the MDAnalysis project. It also entails finding ways to boost participation and involvement from everyone in the MDAnalysis community and to also implement the setup of the MDAnalysis newsletter.

Reviewer: ```suggestion
The goal of this project is to develop a comprehensive social media and communications strategy to grow the MDAnalysis user and contributor base. This involves conducting surveys with MDAnalysis users and developers to understand which communication channels they prefer to use to seek information about and engage with the MDAnalysis project. It also entails finding ways to boost participation and involvement from everyone in the MDAnalysis community and to also implement the setup of the MDAnalysis newsletter.
```

## What I Did

> At the start of the internship, I was able to explore MDAnalysis communication channels and document my observations, recommendations, and current analytics for each channel. I conducted keywords analysis of the MDAnalysis mailing list, Discord, and GitHub discussions along with their frequencies to gain insights into common questions and user needs within the MDAnalysis community. I delved into understanding how participants discover MDAnalysis workshops by gathering data from past MDAnalysis workshops. I also created a methods and definitions document to clarify the metrics used in my report and my approach to the tasks.

Reviewer: ```suggestion
At the start of the internship, I was able to explore MDAnalysis communication channels and document my observations, recommendations, and current analytics for each channel. I conducted keywords analysis, along with their frequencies, of the MDAnalysis mailing lists (now archived), [Discord](https://discord.com/channels/807348386012987462/), and [GitHub Discussions forum](https://github.com/MDAnalysis/mdanalysis/discussions) to gain insights into common questions and user needs within the MDAnalysis community. I delved into understanding how participants discover MDAnalysis workshops by gathering data from past MDAnalysis workshops. I also created a methods and definitions document to clarify the metrics used in my report and my approach to the tasks.
```

Reviewer: Minor grammatical suggestions and links to communication channels

Moving forward, I drafted a survey plan, starting with a description of the MDAnalysis target audience and developing research questions to understand what we aim to learn from the survey. From the research questions, I developed the survey questions and answer options, and the Google form to collect data that will inform our communications strategy. Also, I created a survey outreach plan detailing the outreach and advertising channels we plan to use for distributing the survey, the survey participation plan, and the promotional content (blog post, email, and social media post).

> We launched the survey form on August 1st and I sent follow-up reminders on August 15th and August 23rd respectively. While waiting for the survey responses, I developed a content strategy which included the proposed content plan and calendar for MDAnalysis. In the last week of the internship, I analyzed and wrote a summary of the survey results and how it can inform MDAnalysis communication strategy.

Reviewer: ```suggestion
We [launched the survey form](https://www.mdanalysis.org/2024/07/31/survey-announcement/) on August 1st and I sent follow-up reminders on August 15th and August 23rd respectively. While waiting for the survey responses, I developed a content strategy which included the proposed content plan and calendar for MDAnalysis. In the last week of the internship, I analyzed and wrote a summary of the survey results and how it can inform the MDAnalysis communication strategy.
```

## The Current State

> Currently, the survey results have been analyzed, and a summary has been prepared to inform the MDAnalysis communication strategy and a proposed content plan and calendar for MDAnalysis have been created.

Reviewer: ```suggestion
Currently, the initial survey results have been analyzed, and a summary has been prepared to inform the MDAnalysis communication strategy and a proposed content plan and calendar for MDAnalysis have been created.
```

Reviewer: It might also be worth noting that the survey is still open and making another call for participation if folks would like to! Feel free to use some of the info/links from the [survey blog post](https://www.mdanalysis.org/2024/07/31/survey-announcement/) and your promotional content to write this

> Currently, the survey results have been analyzed, and a summary has been prepared to inform the MDAnalysis communication strategy and a proposed content plan and calendar for MDAnalysis have been created.

Reviewer: ```suggestion
Currently, the initial survey results have been analyzed, and a summary has been prepared to inform the MDAnalysis communication strategy and a proposed content plan and calendar for MDAnalysis have been created.
```

## What's Left to Do

> The next steps involve implementing the communications strategy based on the survey results and making adjustments to the strategy as needed. The survey results will guide the selection of communication channels and content types that will best engage the MDAnalysis community. Additionally, launching the MDAnalysis newsletter is another important part of the project. Beyond the internship, an ongoing effort will be needed to sustain the implementation of the communications strategy to grow MDAnalysis user and contributor’s base.

Reviewer: ```suggestion
The next steps involve implementing the communications strategy based on the survey results and making adjustments to the strategy as needed. The survey results will guide the selection of communication channels and content types that will best engage the MDAnalysis community. Additionally, launching the MDAnalysis newsletter is another important part of the project. Beyond the internship, an ongoing effort will be needed to sustain the implementation of the communications strategy to grow MDAnalysis's user and contributor base.
```

> ## Links to Project Deliverables

Reviewer: I think we should delete this whole section, as the blog post is the only thing that has been (and really should be at this point) widely shared with the community.

Author: Alright. Noted

- [Survey Promotion Content](https://docs.google.com/document/d/1SZIDpFcvfh6yN7pfFYm8UGm3DP7I4w-6BGzME8quGqg/edit?usp=drive_link)
- [MDAnalysis Community Survey Blog Post](https://www.mdanalysis.org/2024/07/31/survey-announcement/)
- [Content Strategy Docs](https://docs.google.com/spreadsheets/d/1p53ojMr3knXymFmeCvgPkud3oOREAymmBav2WKmLvMI/edit?usp=drive_link)
- [Proposed Content Calendar](https://docs.google.com/document/d/1qNhPhJjpAVecqy3pEr50AfQDj1MPdeYFjnFA-T6tbwI/edit?usp=sharing)
- [Events/ Workshop Calendar](https://docs.google.com/document/d/1WNOaAmvAy4A51mJo-EgHlxt6ZBDndBJo0M_ddqEQDi8/edit?usp=drive_link)
- [Survey Analysis Summary](https://docs.google.com/document/d/1kHNvyK0khwu4UBd1mU5v7YiyH6tpWpUlS22Ohzg6GfY/edit?usp=sharing)

## Challenges

One of the main challenges I encountered was drafting research and survey questions that would give the feedback needed to inform MDAnalysis communication strategy. It was important to ask the right questions to gather meaningful feedback from respondents. I was able to navigate this phase with the guidance and valuable input from my mentors and the MDAnalysis leadership.

> Another challenge that I faced was ensuring active participation from the community in the survey. Beyond the initial outreach, my mentor and I sent follow-up reminders, and the survey was promoted during the MDAnalysis 2024 User Group Meeting. While the survey did receive a reasonable number of responses, broader participation from the community could have provided even more meaningful insights.

Reviewer: ```suggestion
Another challenge that I faced was ensuring active participation from the community in the survey. Beyond the initial outreach, my mentor and I sent follow-up reminders, and the survey was promoted during the [MDAnalysis 2024 User Group Meeting](https://www.mdanalysis.org/pages/ugm2024/). While the survey did receive a reasonable number of responses, broader participation from the community could have provided even more meaningful insights.
```

## Lessons Learned

Here are some of the lessons I learned during the internship:
- Prioritize tasks/activities that directly contribute to the project goals. This will help you to stay focused and ensure the overall project goals are achieved.
- Before drafting survey questions, it’s important to first identify the key research questions.
- Learn to embrace and implement feedback from your mentors.
- Breakdown big goals into smaller, achievable weekly tasks. It helped me to stay on track and clearly identify the next steps.
- Communicate with your mentors. Don’t hesitate to ask questions and seek clarity when needed.
- Plan for multiple rounds of feedback and revisions when setting project timelines.
- Document your work and progress when working on a project.
- Be proactive.

## Acknowledgements

> I would like to extend my sincere gratitude to the Outreachy organizers for this incredible opportunity to contribute to the MDAnalysis project.

Reviewer: ```suggestion
I would like to extend my sincere gratitude to the [Outreachy](https://www.outreachy.org/) organizers for this incredible opportunity to contribute to the MDAnalysis project.
```

> A special thanks to the MDAnalysis leadership team and my mentors, Jenna Swarthout Goddard (@jennaswa) and Micaela Matta (@micaela-matta), for their guidance and feedback throughout the internship. This experience has provided me with new skills I will carry forward in my future endeavors.

Reviewer: ```suggestion
A special thanks to the [MDAnalysis leadership team](https://www.mdanalysis.org/about/#mdanalysis-core-developers) and my mentors, Jenna Swarthout Goddard (@jennaswa) and Micaela Matta (@micaela-matta), for their guidance and feedback throughout the internship. This experience has provided me with new skills I will carry forward in my future endeavors.
```

> Finally, I want to express my gratitude to the entire MDAnalysis community for their participation in the survey and for helping make this project a success.

Reviewer: ```suggestion
Finally, I want to express my gratitude to the entire MDAnalysis community for their participation in the survey and for helping make this project a success.

— @adetutudeborah 
```

Reviewer: Good to sign the blog post so it's clear you wrote it :)
